pages/game.html

```diff
@@ -27,6 +27,7 @@
         </nav>
       </header>
       <div id="marquee-container">
+        <!-- todo: 'marquee' is obsolete: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/marquee -->
         <marquee behavior="slide" direction="down" style="width: 100%" id="game-log">Lorem ipsum, dolor sit amet
```

Comment:
Read about it here: [The Marquee element (Obsolete)](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/marquee)

![2019-06-27_01-28-52](https://user-images.githubusercontent.com/30473157/60242280-ecee8f00-987a-11e9-96b4-87bfd5c33b03.png)

You might be able to use CSS animations, instead.